### PR TITLE
Memory game board

### PR DESCRIPTION
**Pull request by tmansuroglu**
#22 this should be merged after MemoryGameCards component

### CODE REVIEW

**Review by louisrli — APPROVED** [review]
isn't confetti in a separate pr already

**Comment by tmansuroglu** [comment]
> isn't confetti in a separate pr already

oh yes. I got confused for a moment

File: src/components/memorygame/MemoryGameCards/index.jsx [hunk 1]
```diff
@@ -52,6 +53,10 @@ const MemoryGameBoard = ({
   // Triggered when user clicks on a card(selectedCards change)
   // checks selected cards to see if they are a match
   useEffect(() => {
+    invariant(
+      selectedCards.length <= NUM_REQUIRED_MATCHES,
+      "'SelectedCards.length' can't be more than 'NUM_REQUIRED_MATCHES'"
```
Comment (louisrli) [inline src/components/memorygame/MemoryGameCards/index.jsx]: you dont need to use the single quotes

Comment (tmansuroglu) [reply src/components/memorygame/MemoryGameCards/index.jsx]: PTAL

**Review by ibrahimaljarad — APPROVED** [review]

**Review by louisrli — APPROVED** [review]

**Comment by tmansuroglu** [comment]
@Ammar-64 can you merge?

**Pull request merged**